src/StoreKeeper/WooCommerce/B2C/Exports/OrderExport.php

```diff
@@ -394,6 +414,51 @@ public static function convertWooCommerceToStorekeeperOrderStatus($wc_order_stat
         }
     }
 
+    /**
+     * @param WC_Order $databaseOrder - Order items to compare from
+     * @param $backofficeOrder - Order items to compare to
+     */
+    public function checkOrderDifference(WC_Order $databaseOrder, $backofficeOrder): bool
+    {
+        $hasDifference = false;
+        $databaseOrderItems = $this->getOrderItems($databaseOrder);
+        $backofficeOrderItems = $backofficeOrder['order_items'];
+
+        foreach ($databaseOrderItems as $databaseOrderItem) {
+            $sku = $databaseOrderItem['sku'];
+            if (!in_array($sku, array_column($backofficeOrderItems, 'sku'), true)) {
+                $hasDifference = true;
+                break;
+            }
+
+            $quantity = $databaseOrderItem['quantity'];
+            $pricePerUnit = $databaseOrderItem['ppu_wt'];
+
+            $databaseOrderItemSet = [$sku, $quantity, $pricePerUnit];
+
+            // Check if any set matches in backoffice order items
+            $hasMatch = false;
+            foreach ($backofficeOrderItems as $backofficeOrderItem) {
+                $backofficeOrderItemSet = [$backofficeOrderItem['sku'], $backofficeOrderItem['quantity'], $backofficeOrderItem['ppu_wt']];
+                if ($backofficeOrderItemSet === $databaseOrderItemSet) {
+                    $hasMatch = true;
+                }
+            }
+
+            if (!$hasMatch) {
+                $hasDifference = true;
+                break;
+            }
+        }
```

Comment:
```suggestion

	$databseSet = [];
        foreach ($databaseOrderItems as $databaseOrderItem) {
            $databseSet[] = ((int)$databaseOrderItem['quantity']).'|'.round($databaseOrderItem['ppu_wt'],2).'|'.$databaseOrderItem['sku'];
        }
        
	$backofficeSet = [];
        foreach ($backofficeOrderItems as $backofficeOrderItem) {
            $backofficeSet[] = ((int)$backofficeOrderItem['quantity']).'|'.round($backofficeOrderItem['ppu_wt'],2).'|'.$backofficeOrderItem['sku'];
        }
        
        sort($databseSet);
        sort($backofficeSet);
        
        $hasDifference = $databseSet !== $backofficeSet;
```

That's what I mean with set comparing.